tests/apollo/test_skvbc_preexecution.py

```diff
@@ -366,6 +366,30 @@ async def test_alternate_f_isolated(self, bft_network, tracker):
             last_block = await tracker.get_last_block_id(read_client)
             assert last_block > start_block
 
+    @with_trio
+    @with_bft_network(start_replica_cmd)
+    @verify_linearizability(pre_exec_enabled=True)
```

Comment:
let's explicitlhy state that `no_conflicts=False` here

Reply:
done